pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -1381,6 +1416,28 @@ public static void verifyHybridTableConfigs(String rawTableName, TableConfig off
     }
   }
 
+  public static void checkForDuplicates(List<String> columns) {
```

Comment:
public static Set<String> findDuplicates(List<String> list) {
        Set<String> seen = new HashSet<>();
        Set<String> duplicates = new HashSet<>();

        for (String item : list) {
            if (!seen.add(item)) {
                duplicates.add(item);
            }
        }

        return duplicates;
    }